File: branches/pre-scala-migration/src/java/org/codehaus/testdox/intellij/actions/ToggleQuickDoxAction.java
package org.codehaus.testdox.intellij.actions;

import com.intellij.openapi.actionSystem.AnActionEvent;
import org.codehaus.testdox.intellij.IconHelper;

public class ToggleQuickDoxAction extends BaseAction {

    public ToggleQuickDoxAction() {
        super("Show QuickDox", "Shows TestDox in a tooltip", IconHelper.getIcon(IconHelper.DOX_ICON));
    }

    public void actionPerformed(AnActionEvent event) {
        actionEvents.getTestDoxController(event).toggleQuickDox();
    }
}

File: trunk/src/main/scala/org/codehaus/testdox/intellij/actions/ToggleQuickDoxAction.scala
package org.codehaus.testdox.intellij.actions

import com.intellij.openapi.actionSystem.AnActionEvent
import org.codehaus.testdox.intellij.Icons._
import org.codehaus.testdox.intellij.Icons

class ToggleQuickDoxAction extends BaseAction("Show QuickDox", "Shows TestDox in a tooltip", getIcon(Icons.DOX_ICON)) {

  def actionPerformed(event: AnActionEvent) { actionEvents.getTestDoxController(event).toggleQuickDox() }
}
